request_listener/curl.go

```diff
@@ -0,0 +1,67 @@
+package request_listener
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+)
+
+func CurlLogDumper(request *http.Request) {
+	if request == nil {
+		return
+	}
+	cmd, err := GetCurlCommand(request)
+	if err != nil {
+		log.Println("cannot print curl command", err)
+		return
+	}
+	log.Println(cmd)
```

Comment:
what if we want to log it in a specific way ? (i.e. with another logger)

Reply:
you can:
- implement your own request listener
- use `log.SetOutput(...)`

i want to keep as light as possible for the moment

Reply:
OK I see 👍